spec/core/finalization/routes/getFinalizationProofAtEpoch.yml

```diff
@@ -1,7 +1,7 @@
 tags:
   - Finalization routes
 summary: Get finalization proof
-description: Gets finalization proof at the given epoch.
+description: Gets finalization proof for the highest height associated with the given epoch.
```

Comment:
```suggestion
description: Gets finalization proof for the greatest height associated with the given epoch.
```

Reply:
up to you,  i just copied core's description here

Reply:
Haha, well, "highest height" sounds bad (in my head) and it's easy to fix so... fixed!